packages/shared/src/contexts/payment/StoreKit.tsx

```diff
@@ -70,6 +71,55 @@ export type PurchaseEvent = {
   detail?: string;
 };
 
+const getApplePlusPricing = (metadata: ProductPricingMetadata[]) => {
+  const response = promisifyEventListener<
+    ProductPricingPreview[],
+    IAPProduct[] | string
+  >('iap-products-result', (event) => {
+    const productsRaw = !isNullOrUndefined(event?.detail) ? event.detail : [];
+
+    // Remove JSON parsing once usage of App v1.8 is low
+    const products: IAPProduct[] =
+      typeof productsRaw === 'string' ? JSON.parse(productsRaw) : productsRaw;
```

Comment:
I'd say v1.9 adoption is quite high, so we can remove this part. @idoshamun concur?
![image](https://github.com/user-attachments/assets/82ef745c-ee37-4603-89d6-8b8e300629f7)

Reply:
Sounds good!

Reply:
Updated!